fcrepo-kernel/src/main/java/org/fcrepo/kernel/observer/FedoraEvent.java

```diff
@@ -59,6 +65,7 @@
      */
     public FedoraEvent(final Event e) {
         checkArgument(e != null, "null cannot support a FedoraEvent!");
+        eventID = pidMinter.mintPid();
```

Comment:
@ajs6f, in your CDI scenario above, would `new FedoraEvent(...)` be called directly? In other words, how are instances created? It seems like calling `new` would circumvent any container management.